code/go/0chain.net/blobbercore/handler/helper.go

```diff
@@ -0,0 +1,17 @@
+package handler
+
+import (
+	"time"
+
+	"github.com/0chain/blobber/code/go/0chain.net/core/common"
+)
+
+func checkValidDate(s string) error {
```

Comment:
how about Timezone? it will be an issue if client send date with local time zone.